src/widget/image.rs

```diff
@@ -0,0 +1,139 @@
+//! Displays image to your users.
+
+use crate::{
+    Style, Node, Element, MouseCursor, Layout, Hasher, Widget,
+    Rectangle, Point,
+};
+
+use std::hash::Hash;
+
+/// A widget that displays an image.
+/// 
+/// It implements [`Widget`] when the associated [`core::Renderer`] implements
+/// the [`image::Renderer`] trait.
+///
+/// [`Widget`]: ../../core/trait.Widget.html
+/// [`core::Renderer`]: ../../core/trait.Renderer.html
+/// [`image::Renderer`]: trait.Renderer.html
+/// # Example
+///
+/// ```
+/// use iced::Image;
+///
+/// let image = Image::new("image");
```

Comment:
I would probably write something like:

```rust
# let my_image_handle = String::from("some_handle"); // This line will be hidden in docs
let image = Image::new(my_image_handle);
```

That way, we remark the fact that the handle is generic.